lib/utils/__tests__/isStandardSyntaxAtRule.test.js

```diff
@@ -1,136 +1,112 @@
 'use strict';
 
 const isStandardSyntaxAtRule = require('../isStandardSyntaxAtRule');
-const less = require('postcss-less');
 const postcss = require('postcss');
-const sass = require('postcss-sass');
-const scss = require('postcss-scss');
+const postcssLess = require('postcss-less');
+const postcssSass = require('postcss-sass');
+const postcssScss = require('postcss-scss');
 
 describe('isStandardSyntaxAtRule', () => {
 	it('non nested at-rules without quotes', () => {
-		atRules('@charset UTF-8;', (atRule) => {
-			expect(isStandardSyntaxAtRule(atRule)).toBeTruthy();
-		});
+		expect(isStandardSyntaxAtRule(atRules('@charset UTF-8;')[0])).toBeTruthy();
```

Comment:
What do you think about adding:

```js
function atRule(code) {
    return atRules(code)[0];
}
```

It will simplify all test cases:

```suggestion
		expect(isStandardSyntaxAtRule(atRule('@charset UTF-8;'))).toBeTruthy();
```

Reply:
Thank you for the feedback! Make sense. 👍 
Fixed via c88a46d2